### ISSUE TITLE
Zoom icon doesn't load for Photo Essay photo galleries

### ISSUE THREAD

**Issue by rebecca-patton** [wvuweb/wvu-design-system-v2#81]
### Steps to reproduce the issue

Create a page using the Photo Essay template and add a gallery to the [page](https://photoessay.sandbox.wvu.edu/photo-essays/2023/07/19/its-a-photo-essay)

### Results

The zoom icon in the upper right of each image in the gallery doesn't load (black square instead)

### Expected results

DS2 [example](https://designsystem.wvu.edu/components/photos-and-videos/photo-gallery)

### Fix

I added the parvus-gallery--custom JS to the component in my [theme](https://bitbucket.org/wvudigital/wvu-magazine-v4/commits/ff20494a2753ae771ca0f88248d5d77fa04fa800).

**Comment by adamjohnson** [comment]
The reason this is happening is because @adamglenn has set `lightboxIndicatorIcon` to an empty string `''` [in this component](https://github.com/wvuweb/wvu-design-system-v2/blob/master/views/components/wvu-photo-set/_wvu-photo-set--v1.html#L74). Learn more in the [Parvus docs](https://github.com/deoostfrees/Parvus#options).

Was this a design decision? 

If it was unintentional, we can simply remove that entire line (`lightboxIndicatorIcon: '',`) and the icons will come back.

While we're at it, `fileTypes: /.*/i ` is no longer needed as of Parvus v2.4.0 and can be removed.

**Comment by adamglenn** [comment]
I set it to an empty string thinking it might remove the icon entirely. When I saw it still kept the gray box there, I meant to go back and figure out what I wanted to do with it, just never got around to it. The reason I wanted it gone was because it never made sense to me as an indicator that it would expand to a larger image and I was trying to mimic the example Matt Sunday gave us for a photo essay, which doesn't include icons on the photos. I can either do what you suggested, AJ, or just add the icon back in (if people think it is helpful). I doubt Matt will care either way.
